source/Cosmos.Build.Builder/CosmosTask.cs

```diff
@@ -319,8 +319,8 @@ private void Restore(string project)
 
       Section("Publish Tools");
       Publish(Path.Combine(mSourcePath, "Cosmos.Build.MSBuild"), Path.Combine(xVsipDir, "MSBuild"));
-      Publish(Path.Combine(mSourcePath, "IL2CPU"), Path.Combine(xVsipDir, "IL2CPU"));
-      Publish(Path.Combine(mSourcePath, "XSharp.Compiler"), Path.Combine(xVsipDir, "XSharp"));
+      Publish(Path.Combine(mSourcePath, "../../IL2CPU"), Path.Combine(xVsipDir, "Cosmos.IL2CPU"));
+      Publish(Path.Combine(mSourcePath, "../../XSharp.Compiler"), Path.Combine(xVsipDir, "Cosmos.XSharp"));
```

Comment:
"XSharp.Compiler" should be changed to "XSharp.XSC". Also, why add "Cosmos." ? It will break the setup.